pkg/objectupdate/rte/rte.go

```diff
@@ -109,6 +110,20 @@ func DaemonSetHashAnnotation(ds *appsv1.DaemonSet, cmHash string) {
 	template.Annotations[hash.ConfigMapAnnotation] = cmHash
 }
 
+func DaemonSetArgs(ds *appsv1.DaemonSet) error {
```

Comment:
minor comment: maybe it is because I am a little bit out of context but function name could be a little more expressive.